hypha/settings/base.py

```diff
@@ -201,12 +203,34 @@
 # Database
 # https://docs.djangoproject.com/en/stable/ref/settings/#databases
 
-DATABASES = {
-    'default': dj_database_url.config(
-        conn_max_age=600,
-        default=f"postgres:///{APP_NAME}"
-    )
-}
+if 'SALESFORCE_INTEGRATION' in env and env['SALESFORCE_INTEGRATION']:
+    DATABASES = {
+        'default': dj_database_url.config(
+            conn_max_age=600,
+            default=f"postgres:///{APP_NAME}"
+        ),
+        'salesforce': {
+            'ENGINE': 'salesforce.backend',
+            'CONSUMER_KEY': env['CONSUMER_KEY'],
+            'CONSUMER_SECRET': env['CONSUMER_SECRET'],
+            'USER': env['SALESFORCE_USER'],
+            'PASSWORD': env['SALESFORCE_PASSWORD'],
+            'HOST': env['SALESFORCE_LOGIN_URL']
+        }
+    }
+
+    SALESFORCE_QUERY_TIMEOUT = (30, 30)  # (connect timeout, data timeout)
+
+    DATABASE_ROUTERS = [
+        "salesforce.router.ModelRouter"
+    ]
+else:
+    DATABASES = {
+        'default': dj_database_url.config(
+            conn_max_age=600,
+            default=f"postgres:///{APP_NAME}"
+        )
+    }
 
```

Comment:
I would prefer to put all the Salesforce settings in a separate section. Something like this.

~~~~
# Salesforce integration

if env.get('SALESFORCE_INTEGRATION', 'false').lower().strip() == 'true':
    DATABASES = {
        **DATABASES,
        'salesforce': {
            'ENGINE': 'salesforce.backend',
            'CONSUMER_KEY': env.get('SALESFORCE_CONSUMER_KEY', ''),
            'CONSUMER_SECRET': env.get('SALESFORCE_CONSUMER_SECRET', ''),
            'USER': env.get('SALESFORCE_USER', ''),
            'PASSWORD': env.get('SALESFORCE_PASSWORD', ''),
            'HOST': env.get('SALESFORCE_LOGIN_URL', '')
        }
    }

    SALESFORCE_QUERY_TIMEOUT = (30, 30)  # (connect timeout, data timeout)

    DATABASE_ROUTERS = [
        "salesforce.router.ModelRouter"
    ]
~~~~